.github/workflows/os-runner.yml

```diff
@@ -0,0 +1,40 @@
+name: OS Runner Image
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    paths:
+      - 'util/os-runner/**'
+
+env:
+  IMAGE_NAME: os-runner
+  IMAGE_TAG: latest
+  IMAGE_ID: ghcr.io/${{ github.repository_owner }}/$IMAGE_NAME
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    strategy:
+      fail-fast: false
+      matrix:
+        architecture:
+          - linux/amd64
+
+    steps:
+      - uses: actions/checkout@v3
+
```

Comment:
Might also need to setup qemu: https://github.com/edi33416/docker-buildx-ci/blob/main/.github/workflows/ci.yml#L15-L16

Reply:
Done